applications/FluidDynamicsApplication/python_scripts/apply_outlet_process.py

```diff
@@ -128,15 +128,17 @@ def _ComputeOutletCharacteristicVelocity(self):
         for node in self.outlet_model_part.GetCommunicator().LocalMesh().Nodes:
             vnode = node.GetSolutionStepValue(KratosMultiphysics.VELOCITY)
             outlet_avg_vel_norm += math.sqrt(vnode[0]*vnode[0] + vnode[1]*vnode[1] + vnode[2]*vnode[2])
-            outlet_avg_vel_norm /= len(self.outlet_model_part.GetCommunicator().LocalMesh().Nodes)
```

Comment:
i'll accept this for now, however it should be definitely be implemented in c++ (it is a good candidate for a function in variable utils)